Sources/NIOHTTP2/InboundEventBuffer.swift

```diff
@@ -65,3 +70,9 @@ extension InboundEventBuffer: CustomStringConvertible {
         return "InboundEventBuffer { \(self.buffer) }"
     }
 }
+
+#if swift(>=5.5) && canImport(_Concurrency)
+extension InboundEventBuffer: @unchecked Sendable {
```

Comment:
This type is very deliberately not `Sendable` and shouldn't be. Can we revert the change here?